File: tests/testdata/codegen/unknown-invoke-pp/go/unknown-invoke.go
package main

import (
	"github.com/khulnasoft/codeinfra-unknown/sdk/go/unknown"
	"github.com/khulnasoft/codeinfra-unknown/sdk/go/unknown/eks"
	"github.com/khulnasoft/codeinfra/sdk/v3/go/codeinfra"
)

func main() {
	codeinfra.Run(func(ctx *codeinfra.Context) error {
		data, err := unknown.GetData(ctx, map[string]interface{}{
			"input": "hello",
		}, nil)
		if err != nil {
			return err
		}
		_, err = eks.ModuleValues(ctx, map[string]interface{}{}, nil)
		if err != nil {
			return err
		}
		ctx.Export("content", data.Content)
		return nil
	})
}

File: tests/testdata/codegen/unknown-invoke-pp/dotnet/unknown-invoke.cs
using System.Collections.Generic;
using System.Linq;
using Codeinfra;
using Unknown = Codeinfra.Unknown;

return await Deployment.RunAsync(() => 
{
    var data = Unknown.Index.GetData.Invoke(new()
    {
        Input = "hello",
    });

    var values = Unknown.Eks.ModuleValues.Invoke();

    return new Dictionary<string, object?>
    {
        ["content"] = data.Content,
    };
});

